plugins/com.google.cloud.tools.eclipse.integration.appengine/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,21 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Integration tests for App Engine functionality
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.integration.appengine
+Bundle-Version: 0.1.0.qualifier
+Bundle-Vendor: Google Inc.
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Require-Bundle: com.google.cloud.tools.eclipse.swtbot;bundle-version="0.1.0",
```

Comment:
At many other bundles we don't specify versions for those that are part of this project (also the deploy bundle has no version set a few lines below this). Shouldn't it be consistent?
Would the tycho-version-plugin update these versions as well along with the ones in plugin/feature manifests?